scipy/stats/_continuous_distns.py

```diff
@@ -6434,6 +6434,357 @@ def _stats(self, a):
     def _entropy(self, a):
         return 1 - 1.0/a - np.log(a)
 
+    def fit(self, data, *args, **kwds):
+        data, fshape, floc, fscale = _check_fit_input_parameters(self, data,
+                                                                 args, kwds)
+        def shape_universal(data, loc, scale):
+            mask = data != loc
+            return -len(data) / np.sum(np.log((data[mask] - loc)/scale))
+        
+        
+        # First, attempt to fit under assumption that `shape <= 1`. If the shape is 
+        # greater than 1, then we need to fit under the assumption that `shape > 1`.
+        def universal_slte1(data):
+            print(f"shape, loc, scale : {fshape, floc, fscale}")
+
+            print("universal_slte1")
+            loc = floc if floc is not None else np.nextafter(data.min(), -np.inf)
+            scale = fscale or np.nextafter(np.ptp(data), np.inf)
+            shape = fshape or shape_universal(data, loc, scale)
```

Comment:
The `.nextafter` is required for the scale not to explode. I tried passing in the non-modified scale to the shape function, and then doing `nextafter` on the scale after that, but no luck with that either. (Like this: 

```
            loc = floc if floc is not None else np.nextafter(data.min(), -np.inf)
            scale = fscale or np.ptp(data)
            shape = fshape or shape_universal(data, loc, scale)
            scale = np.nextafter(scale, np.inf)
```